.devcontainer/docker-compose.yml

```
@@ -1,97 +1,98 @@
+    depends_on:
+      - database
+    tmpfs:
+      - /tmp
 
 volumes:
-  gem_cache:
-  postgres:
-  node_modules:
+  bundle_cache:
+  postgres_data:
+  node_modules:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)